src/draw/sw/lv_draw_sw_gradient.h

```diff
@@ -44,6 +44,8 @@ typedef lv_color_t lv_grad_color_t;
  * @param dsc       The gradient descriptor to use
  * @param range     The range to use in computation.
  * @param frac      The current part used in the range. frac is in [0; range]
+ * @param color_out ???
```

Comment:
`color_out` is the calculated gradient color
`opa_out` is the corresponding opacity.

Reply:
@XuNeo Thank you!  I have updated these 2 for a subsequent PR.  :-)